app/models/media_object.rb

```diff
@@ -156,9 +156,13 @@ def collection= co
   # Sets the publication status. To unpublish an object set it to nil or
   # omit the status which will default to unpublished. This makes the act
   # of publishing _explicit_ instead of an accidental side effect.
-  def publish!(user_key)
+  def publish!(user_key, validate: false)
```

Comment:
Sorry, I think I messed up the default value.  In order to maintain backwards compatibility we should validate by default.
```suggestion
  def publish!(user_key, validate: true)
```